util/crs-rules-check/rules-check.py

```diff
@@ -569,6 +569,29 @@ def check_pl_consistency(self):
                     severity  = None     # severity
                     has_nolog = False    # rule has nolog action
 
+def removecomments(data):
+    _data = []  # new structure by lines
+    lines = [l.strip("\n") for l in data.split("\n")] # split the old content
```

Comment:
`split` places one entry in the array per delimiter, so there won't be any `\n` left.
```suggestion
    lines = data.split("\n")
```